src/app/Library/CrudPanel/Traits/Access.php

```diff
@@ -24,7 +24,7 @@ public function allowAccess($operation)
     /**
      * Disable the access to a certain operation, or the current one.
      *
-     * @param  array  $operation  [description]
+     * @param  string|array  $operation  [description]
      * @return [type] [description]
```

Comment:
🔴

Reply:
?

Reply:
What's the `[description]`, and what return type is `[type]` ?